core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -848,8 +847,7 @@ object SnappyContext extends Logging {
     TOPK_SOURCE -> TOPK_SOURCE_CLASS,
     SOCKET_STREAM_SOURCE -> classOf[SocketStreamSource].getCanonicalName,
     FILE_STREAM_SOURCE -> classOf[FileStreamSource].getCanonicalName,
-    KAFKA_STREAM_SOURCE -> classOf[KafkaStreamSource].getCanonicalName,
-    DIRECT_KAFKA_STREAM_SOURCE -> classOf[DirectKafkaStreamSource].getCanonicalName,
+    KAFKA_STREAM_SOURCE -> classOf[DirectKafkaStreamSource].getCanonicalName,
```

Comment:
We need documentation and release notes that "kafka_stream" now means "directkafka_stream". If there are any existing users of "kafka_stream" then we cannot change this rather will need to support both like before.

Reply:
@sumwale Yes, there will be doc change also there is a syntax change in Kafka param as well. We need to update the docs accordingly.